index.html

```diff
@@ -61,5 +67,168 @@ <h2 class="yoda">
+          <img src="Snapshoot Portfolio(2).svg" alt="" class="gridpicture" />
+          <h2 class="tonic">Tonic</h2>
+          <ul class="grid-list">
+            <li class="canopy-list">CANOPY</li>
+            <li class="back-list">Back End Dev</li>
+            <li class="year-list">2015</li>
+          </ul>
+          <p class="gridtext">
+            A daily selection of privately <br />
+            personalized reads; no accounts or <br />
+            sign-ups required.
+          </p>
+          <ul class="grid-list2">
+            <li class="list2">HTML</li>
+            <li class="list2">CSS</li>
+            <li class="list2">Javascript</li>
+          </ul>
+          <h3 class="seeproject">See Project</h3>
+        </div>
+
+        <div class="grid-container">
+          <img src="Snapshoot Portfolio(3).svg" alt="" class="gridpicture" />
+          <h2 class="tonic">
+            Multi-Post <br />
+            Stories
+          </h2>
+          <ul class="grid-list">
+            <li class="canopy-list">CANOPY</li>
+            <li class="back-list">Back End Dev</li>
+            <li class="year-list">2015</li>
+          </ul>
+          <p class="gridtext">
+            A daily selection of privately <br />
+            personalized reads; no accounts or <br />
+            sign-ups required.
+          </p>
+          <ul class="grid-list2">
+            <li class="list2">HTML</li>
+            <li class="list2">CSS</li>
+            <li class="list2">Javascript</li>
+          </ul>
+          <h3 class="seeproject">See Project</h3>
+        </div>
+      </div>
+    </section>
+    <section class="about">
+      <div class="about-container">
+        <h2 class="yoda">About<br />Myself</h2>
+        <p class="intro-text">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+        <p class="connect">LET'S CONNECT</p>
+
+        <ul class="social-media">
+          <li class="social-icon">
+            <a href="">
+              <img src="images/github.svg" alt="github" />
+            </a>
+          </li>
+          <li class="social-icon">
+            <a href="">
+              <img src="images/Linkedin.svg" alt="Linkedin" />
+            </a>
+          </li>
+          <li class="social-icon">
+            <a href="">
+              <img src="images/victory.svg" alt="victory" />
+            </a>
+          </li>
+          <li class="social-icon">
+            <a href="#">
+              <img src="images/twitter.svg" alt="twitter" />
+            </a>
+          </li>
+          <li class="social-icon">
+            <a href="">
+              <img src="images/microverse.svg" alt="Meta" />
+            </a>
+          </li>
+        </ul>
+
+        <p class="resume">Get My Resume</p>
```

Comment:
- [ ] Kindly use the capital letter in the correct words to be similar to the Figma design 👇🏽 

![image](https://github.com/CrackedYoda/MyPortfolio/assets/58642949/3e4d0473-43ee-48c3-88f1-d7d3b86957a9)